cypress/integration/templates_test.js

```diff
@@ -0,0 +1,45 @@
+describe("Templates Test", function() {
+    beforeEach(function() {
+      cy.visit("/");
+    });
+
+    it("can be accessed from the navigation tab", function() {
+        cy.openTemplatesTab();
+        cy.url().should("include", "/templates");
+    });
+
+    it("can be accessed from the main page", function() {
+        cy.contains("More queries here").click();
+        cy.url().should("include", "/templates");
+    });
+
+    it("can filter templates by search", function() {
+        cy.openTemplatesTab();
+        cy.url().should("include", "/templates");
+        cy.get('input[class="form-control input-lg"]').type("Organism{enter}",{delay:100});
+        cy.get('#Organism_Protein').should("have.id","Organism_Protein");
+    });
+
+    it("can search templates by categories", function() {
+        cy.openTemplatesTab();
+        cy.url().should("include","/templates");
+        cy.get("div[class=grid-1]").should("have.length","3");
+        cy.contains("Genomics").click();
+        cy.get("div[class=grid-1]").should("have.length","1");
```

Comment:
`div[class=grid-1]` is more likely to change than `.template`. Have you tried selecting for `.template` and checking length of that?